File: niuniu-take-out-springboot3/server/src/main/java/common/niuniu/service/WorkSpaceService.java
package common.niuniu.service;

import common.niuniu.vo.BusinessDataVO;
import common.niuniu.vo.DishOverViewVO;
import common.niuniu.vo.OrderOverViewVO;
import common.niuniu.vo.SetmealOverViewVO;

import java.time.LocalDateTime;

public interface WorkSpaceService {
    BusinessDataVO getBusinessData(LocalDateTime begin, LocalDateTime end);

    OrderOverViewVO getOrderOverView();

    DishOverViewVO getDishOverView();

    SetmealOverViewVO getSetmealOverView();
}

File: niuniu-take-out-kotlin/server/src/main/kotlin/common/niuniu/service/WorkSpaceService.kt
package common.niuniu.service

import common.niuniu.vo.BusinessDataVO
import common.niuniu.vo.DishOverViewVO
import common.niuniu.vo.OrderOverViewVO
import common.niuniu.vo.SetmealOverViewVO
import java.time.LocalDateTime

interface WorkSpaceService {
    fun getBusinessData(begin: LocalDateTime, end: LocalDateTime): BusinessDataVO?

    val orderOverView: OrderOverViewVO?

    val dishOverView: DishOverViewVO?

    val setmealOverView: SetmealOverViewVO?
}
